pkg/controller/job/job_controller.go

```diff
@@ -440,7 +443,7 @@ func (jm *Controller) getPodsForJob(j *batch.Job) ([]*v1.Pod, error) {
 // syncJob will sync the job with the given key if it has had its expectations fulfilled, meaning
 // it did not expect to see any more of its pods created or deleted. This function is not meant to be invoked
 // concurrently with the same key.
-func (jm *Controller) syncJob(key string) (bool, error) {
+func (jm *Controller) syncJob(key string) (forgetOrNot bool, errMsg error) {
```

Comment:
nit: name them `forget` and `err`. If you are worried about name conflicts, perhaps you can use `rErr`. I think `forget` should be fine to reuse within the function.